src/ocean/text/xml/Document.d

```diff
@@ -2082,6 +2082,95 @@ public class XmlPath(T)
         }
 }
 
+version ( UnitTest )
+{
+    import ocean.core.Test;
+    import ocean.text.xml.DocPrinter;
+}
+
+unittest
+{
+    auto t = new NamedTest("ocean.text.xml.Document");
```

Comment:
Why using a named test?

Reply:
No reason really, I tend to always used named test to get slightly more information on the test. I can remove it though.